packages/graphql-transformer-common/src/util.ts

```diff
@@ -28,4 +28,5 @@ export function toPascalCase(words: string[]): string {
     return formatted.join('')
 }
 
-export const NONE_VALUE = '___xamznone____'
\ No newline at end of file
+export const NONE_VALUE = '___xamznone____'
+export const NONE_INT_VALUE = -2147483648;
```

Comment:
What does this magic number mean

Reply:
Int32.MinValue, we cannot use 0 as a "none value" for numeric default-if-null default values.